README.md

```diff
@@ -112,99 +172,140 @@ The job will appear on the WebUI and you can inspect/restart/kill it.
+is used to submit workflows to the engine.
 
 A workflow is a Graph of “**targets**”.
 
 There are 3 kinds of links between targets:
 
-- *dependencies:* targets that need to be ensured or
+- *dependencies:* targets that need to be ensured to exist or
 run before a target can start,
-- *fallbacks:* targets that will be activated
-if the target fails, and
-- *success-triggers:* targets that will be activated only *after* a target
-succeeds.
+- *fallbacks:* targets that will be activated if the target fails, and
+- *successes:* targets that will be activated only *after* a target succeeds.
 
-Any OCaml program can use the EDSL (script, compiled, or even inside the
-toplevel), see the [documentation of the EDSL API](src/lib/eDSL.mli).
+See the [target](https://github.com/hammerlab/ketrew/blob/master/src/lib/eDSL.mli#L176)
+function documentation for details. Any OCaml program can use the EDSL (script, compiled,
+or even inside the toplevel), see the [documentation of the EDSL API](src/lib/eDSL.mli).
 
 
 ### Example
 
-This example is a “single-target” workflow that runs an arbitrary shell command
-on an [LSF-based](http://en.wikipedia.org/wiki/Platform_LSF) cluster:
+The following script extends the previous shell based example with the capability to send emails upon the success or failure of your command.
 
 ```ocaml
 #use "topfind"
 #thread
 #require "ketrew"
-let run_command_with_lsf cmd =
+
+let run_command_with_daemonize ~cmd ~email =
   let module KEDSL = Ketrew.EDSL in
-  let host =
-    (* `Host.parse` takes an URI and creates a “Host” datastructue: a place to
-       run stuff.  *)
-    KEDSL.Host.parse
-      "ssh://user42@MyLSFCluster/home/user42/ketrew-playground/?shell=bash"
-    (* This one is an SSH host, named `MyLSFCluster`.
-       The directory `/home/user42/ketrew-playground/` will be used by Ketrew
-       to monitor the jobs. *)
-  in
-  let program =
-    (* A “program” is a datastructure representing “extended shell scripts”.
-       `Program.sh` creates one out a shell command. *)
-    KEDSL.Program.sh cmd in
-  let lsf_build_process =
-    (* “build process” is a method for making things:
-       `lsf` creates a datastructure that represents a job running a `program`
-       with the LSF scheduling engine, on the host `host`.  *)
-    KEDSL.lsf
-      ~queue:"normal-people" ~wall_limit:"1:30"
-      ~processors:(`Min_max (1,1)) ~host program
+
+  (* Where to run stuff *)
+  let host = KEDSL.Host.tmp_on_localhost in
+
+  (* A “program” is a datastructure representing an “extended shell script”. *)
+  let program = KEDSL.Program.sh cmd in
+
+  (* A “build process” is a method for making things.
+     
+     In this case, `daemonize` creates a datastructure that represents a job
+     running our program on the host. *)
+  let build_process = KEDSL.daemonize ~host program in
+
+  (* A target that Ketrew will activate after the success of cmd *)
```

Comment:
Updated the comment. Ugh, yea `mail` may not be universal, but (1) it is obvious what it does, (2) it has almost straightforward syntax and (3) it at least it exists on our dev machines giving me hope that it exists on `Linux`